### ISSUE TITLE
no apps displayed

### ISSUE THREAD

**Issue by animalillo** [AndlyticsProject/andlytics#718]
Y get no apps listed, although I have 1 app published

**Comment by willlunniss** [comment]
You are going to have to provide some actual information, at a minimum a logcat output.

**Comment by animalillo** [comment]
I've built the latest tag from the repo (2.6.2) and this is the logcat

[logcat](https://gist.github.com/animalillo/28536f23ccc2129d9153)

**Comment by willlunniss** [comment]
I see some authentication errors, what sort of account do you have, did you grant the needed access when prompted?

**Comment by animalillo** [comment]
Yes, I granted all access requested, maybe what you see is related with the prompt about the app not being registered in google message the authentication prompt shown, but you can see my the only app i have published there in the logs; search for "es.wolfi.running" and you'll see it, that won't be accessed by the app unless access was granted.

**Comment by willlunniss** [comment]
Sorry I missed that bit as was looking on a phone. Can you activate debug mode in https://github.com/AndlyticsProject/andlytics/blob/master/src/com/github/andlyticsproject/console/v2/JsonParser.java and post the extra data. Comment out the xsrf etc... tokens.

**Comment by animalillo** [comment]
Here you go [logcat v2](https://gist.github.com/animalillo/3e12282a48e82e372ab5)
